web_src/js/modules/fomantic/dropdown.js

```diff
@@ -207,7 +207,7 @@ function attachDomEvents(dropdown, focusable, menu) {
       if (!$item) $item = $(menu).find('> .item.selected'); // when dropdown filters items by input, there is no "value", so query the "selected" item
       // if the selected item is clickable, then trigger the click event.
       // we can not click any item without check, because Fomantic code might also handle the Enter event. that would result in double click.
-      if ($item && ($item[0].matches('a') || $item.hasClass('js-aria-clickable'))) $item[0].click();
+      if ($item?.[0]?.matches('a, .js-aria-clickable')) $item[0].click();
```

Comment:
Why `, .`? Are we not looking for a `a.js-aria-clickable`?

Reply:
The original code doesn't

Reply:
Oh I see, I missed that this is `||`.